Food/git-remote-codecommit.lua

```diff
@@ -0,0 +1,50 @@
+local name = "git-remote-codecommit"
+local version = "v1.0.0"
```

Comment:
`version` fields should not include a `v` in the string. Gofish uses this as part of its semantic version check.

Have a look at similar files like [bicep](https://github.com/fishworks/fish-food/blob/main/Food/bicep.lua) on how to accomplish this.

Reply:
I have followed the approach used by bicep. Thanks for letting me know